modules/apps/headless-apio/structure/structure-apio-test/src/testIntegration/java/com/liferay/structure/apio/internal/util/test/StructureRepresentorBuilderHelperTest.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.structure.apio.internal.util.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.dynamic.data.mapping.model.DDMFormField;
+import com.liferay.registry.Registry;
+import com.liferay.registry.RegistryUtil;
+import com.liferay.structure.apio.architect.util.StructureRepresentorBuilderHelper;
+
+import java.lang.reflect.Method;
+
+import java.util.Collection;
+import java.util.Iterator;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Ruben Pulido
+ */
+@RunWith(Arquillian.class)
+public class StructureRepresentorBuilderHelperTest {
+
+	@Test
+	public void testBooleanFieldDataType() throws Exception {
+		DDMFormField ddmFormFieldValue = new DDMFormField(
```

Comment:
ddmFormField

Reply:
Thanks for catching that! Fixed here: https://github.com/nhpatt/liferay-portal/pull/72/